l3kernel/l3bitset.dtx

```diff
@@ -119,40 +121,39 @@
 %         \meta{name2} |=| \meta{index2} |,| \ldots{}
 %      \}
```

Comment:
From `texdoc l3bitset`, indents are missing in this and several more `syntax` envs.

Reply:
That we can deal with post-PR I think